chore(docs): specify syntax highlighting language for code blocks

domust opened the pull request:
Hi,

I've been using this library for the last couple of days and it's great.
It was a bit hard to read documentation without syntax highlighting though, so I've decided to contribute. I've tried to follow the commit message format to the best of my ability.

## Summary by Sourcery

Specify syntax highlighting languages for code blocks in README.md

Documentation:
- Annotate Go code examples with ```go fences for syntax highlighting
- Annotate SQL examples with ```sql fences for syntax highlighting

cnlangzi commented:
@domust  docs should not affect Lint check. It must be an other problem.  I'm really appreciated if you can get some time left to check and fix it. thanks.

domust commented:
@cnlangzi it was the default behavior of golangci-lint, it lints the entire project instead of only the parts that were changed.

How would you like to proceed with the whole merging process? The 2 additional commits that addressed the issues don't follow the project's convention. Will those be squashed on merge or will I have to update the PR to contain only a single commit for it to be merged?

cnlangzi reviewed (approved):
Excellent PR 👍

cnlangzi commented:
> @cnlangzi it was the default behavior of golangci-lint, it lints the entire project instead of only the parts that were changed.
> 
> How would you like to proceed with the whole merging process? The 2 additional commits that addressed the issues don't follow the project's convention. Will those be squashed on merge or will I have to update the PR to contain only a single commit for it to be merged?

- `only_new_issue` seems don't work.  
- Don't worry about commit log, let's squashed them on merging.

cnlangzi commented:
@domust  let's merge it first. It should run the old github action from `main` branch.

domust commented:
@cnlangzi if `only_new_issue` doesn't work, it means that `golangci-lint` is reporting errors incorrectly when `go.sum` is missing entries. `go mod tidy` would fix the issue if that's the cause.
